command/exec.go

```diff
@@ -235,27 +222,21 @@ func (c *subprocessExec) Execute(ctx context.Context, comm client.Communicator,
 	return err
 }
 
-func (c *subprocessExec) runCommand(ctx context.Context, taskID string, proc subprocess.Command, logger client.LoggerProducer) error {
+func (c *subprocessExec) runCommand(ctx context.Context, taskID string, cmd *jasper.Command, logger client.LoggerProducer) error {
 	if c.Silent {
 		logger.Execution().Info("executing command in silent mode")
 	}
 
-	if err := proc.Start(ctx); err != nil {
-		return errors.Wrap(err, "problem starting background process")
-	}
-	pid := proc.GetPid()
-	subprocess.TrackProcess(taskID, pid, logger.System())
-
-	if c.Background {
-		logger.Execution().Infof("started background process with pid %d", pid)
```

Comment:
These log messages are sometimes valuable. Does jasper do something similar?

Reply:
All of this logging happens in the process constructor now and should still be around.